backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -825,41 +825,40 @@ private void handleCancelledSystem() {
         if (stopImageTransferSession(context.entity)) {
             Guid transferingVdsId = context.entity.getVdsId();
 
-            // Verify image is relevant only on upload.
-            if (getParameters().getTransferType() == TransferType.Upload) {
-                isImageVerified = verifyImage(transferingVdsId);
-                if (isImageVerified) {
-                    // We want to use the transferring vds for image actions for having a coherent log when transferring.
-                    setVolumeLegalityInStorage(LEGAL_IMAGE);
-                    if (getDiskImage().getVolumeFormat().equals(VolumeFormat.COW)) {
-                        setQcowCompat(getDiskImage().getImage(),
-                                getStoragePool().getId(),
-                                getDiskImage().getId(),
-                                getDiskImage().getImageId(),
-                                getStorageDomainId(),
-                                transferingVdsId);
-                        imageDao.update(getDiskImage().getImage());
-                    }
+            // Verify image is relevant only on upload
+            if (getParameters().getTransferType() == TransferType.Download) {
+                setAuditLogTypeFromPhase(ImageTransferPhase.FINISHED_SUCCESS);
+                setCommandStatus(CommandStatus.SUCCEEDED);
+            } else if (verifyImage(transferingVdsId)) {
+                // We want to use the transferring vds for image actions for having a coherent log when transferring.
+                setVolumeLegalityInStorage(LEGAL_IMAGE);
+                if (getDiskImage().getVolumeFormat().equals(VolumeFormat.COW)) {
+                    setQcowCompat(getDiskImage().getImage(),
+                            getStoragePool().getId(),
+                            getDiskImage().getId(),
+                            getDiskImage().getImageId(),
+                            getStorageDomainId(),
+                            transferingVdsId);
+                    imageDao.update(getDiskImage().getImage());
                 }
+                setAuditLogTypeFromPhase(ImageTransferPhase.FINISHED_SUCCESS);
+                setCommandStatus(CommandStatus.SUCCEEDED);
+            } else {
+                nextImageStatus = ImageStatus.ILLEGAL;
+                updateEntityPhase(ImageTransferPhase.FINALIZING_FAILURE);
             }
 
             // Finished using the image, tear it down.
             tearDownImage(context.entity.getVdsId(), context.entity.getBackupId());
 
-            setImageStatus(isImageVerified? ImageStatus.OK : ImageStatus.ILLEGAL);
-        }
-
-        if (isImageVerified) {
-            log.info("Image transfer '{}' was successful for {}", getCommandId(), getTransferDescription());
-            setCommandStatus(CommandStatus.SUCCEEDED);
-        } else {
-            updateEntityPhase(ImageTransferPhase.FINALIZING_FAILURE);
+            // Moves Image status to OK or ILLEGAL
```

Comment:
"Image status" --> "image status" or "Image Status".
Add dot at the end.

Reply:
It doesn't add enough value in my opinion, I'd rather not add changes that aren't necessary since this patch may need to be backported